### ISSUE TITLE
Asking for code

### ISSUE THREAD

**Issue by ccmCCMfk** [wurina-github/FGTrans#1]
thanks for your great work,do you have the plan to put it on this website?

**Comment by wurina-github** [comment]
你好，我收到你的邮件啦，我会尽快查看滴

**Comment by wurina-github** [comment]
Yes, I am cleaning up the code and will put it on this website, thank you for your attention

**Comment by ccmCCMfk** [comment]
你好，我是中国科学技术大学一名研一学生，我看您文章单位是大连理工大学，我们组最近也买了五指灵巧手和UR10e，我们可以留个微信交流合作吗，目前我们还在仿真阶段还没开始做实物，后面也许可以一起合作交流共同赶上国外那些组的步伐。





------------------ Original ------------------
From: wurina-github ***@***.***&gt;
Date: Mon,Jun 26,2023 11:54 AM
To: wurina-github/FGTrans ***@***.***&gt;
Cc: ccmCCMfk ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [wurina-github/FGTrans] Asking for code (Issue #1)





 
Yes, I am cleaning up the code and will put it on this website, thank you for your attention

**Comment by wurina-github** [comment]
你好，我收到你的邮件啦，我会尽快查看滴

**Comment by wurina-github** [comment]
请通过邮箱 hswrn@mail.dlut.edu.cn联系，谢谢！

**Comment by ccmCCMfk** [comment]
> 请通过邮箱 [hswrn@mail.dlut.edu.cn](mailto:hswrn@mail.dlut.edu.cn)联系，谢谢！

好的，谢谢你的回复
